index.html

```diff
@@ -255,15 +251,267 @@ <h3><a name="tutorials-twentymins">The twenty minute tour</a></h3>
 
     <h3><a name="tutorials-fa1.2">Deploying an FA1.2 fungible token contract</a></h3>
     <p>
-      Coming soon..
+      The FA1.2 smart contract standard marked a new step in the development possibilities of the Tezos ecosystem by setting specifications for the creation and management of new fugible tokens on the Tezos network. We can now deploy smart contracts to create tokens that can be transferred and exchanged with no loss of value directly on the Tezos blockchain!    </p>
```

Comment:
The first sentence runs on too long. Can you refactor it into smaller, easily readable sentenced?